index.html

```diff
@@ -129,5 +129,26 @@ <h4 class="list-title">Skills</h4>
       </li>
     </ul>
   </section>
+  <section class="contact">
+    <h3 class="contact-header">Get started</h3>
+    <p class="contact-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+    <form class="form" action="https://formspree.io/f/mjvzkaal" method="post">
+      <input type="text" maxlength="30" placeholder="Your name" class="form-input" required />
+      <input type="email" placeholder="Your email" class="form-input" required />
+      <textarea maxlength="500" placeholder="Enter your message here..." class="form-text" required></textarea>
+      <button type="submit" class="form-btn">Start collaboration</button>
+    </form>
```

Comment:
- [ ] When I tried to submit a message I got the following error, kindly fix it. Suggestion: try adding the `name` attribute maybe that will fix it.
![image](https://user-images.githubusercontent.com/50721479/187027720-4c1f77fb-3e4a-45d3-936b-86d42acdfaaa.png)